pkg/trustedresources/verify.go

```diff
@@ -75,7 +93,19 @@ func VerifyTask(ctx context.Context, taskObj v1beta1.TaskObject, k8s kubernetes.
 		ObjectMeta: tm,
 		Spec:       taskObj.TaskSpec(),
 	}
-	verifiers, err := getVerifiers(ctx, k8s)
+	// try to fetch from resource namespace
+	vp, err := tekton.TektonV1alpha1().VerificationPolicies(namespace).List(ctx, metav1.ListOptions{})
```

Comment:
This will increase more requests calls to api server.  I think we should avoid this? 
One way is to remove the new crd and only use configmap to parse the data into the structured policy.
The other is to create a new controller  and converts it to configmap in the reconciler. 
Like https://github.com/sigstore/policy-controller/blob/ba6cdef1d82974440ea2eb23029fe6150bceb265/pkg/reconciler/clusterimagepolicy/clusterimagepolicy.go#L75-L105
wdyt?
@bobcatfish

Reply:
Another idea is to add VerificationPolicyLister into taskrun, pipelinerun reconcilers, will that address this issue?

Reply:
So can we just use lister in this pr, and mark configmap as a TODO?